hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractUpdateSuite.java

```diff
@@ -86,22 +86,24 @@ public boolean canRunConcurrent() {
 
     @Override
     public List<HapiSpec> getSpecsInSuite() {
-        return List.of(
-                updateWithBothMemoSettersWorks(),
-                updatingExpiryWorks(),
-                rejectsExpiryTooFarInTheFuture(),
-                updateAutoRenewWorks(),
-                updateAdminKeyWorks(),
-                canMakeContractImmutableWithEmptyKeyList(),
-                givenAdminKeyMustBeValid(),
-                fridayThe13thSpec(),
-                updateDoesNotChangeBytecode(),
-                eip1014AddressAlwaysHasPriority(),
-                immutableContractKeyFormIsStandard(),
-                updateAutoRenewAccountWorks(),
-                updateStakingFieldsWorks());
+        return List.of(canMakeContractImmutableWithEmptyKeyList());
+        //        return List.of(
+        //                updateWithBothMemoSettersWorks(),
+        //                updatingExpiryWorks(),
+        //                rejectsExpiryTooFarInTheFuture(),
+        //                updateAutoRenewWorks(),
+        //                updateAdminKeyWorks(),
+        //                canMakeContractImmutableWithEmptyKeyList(),
+        //                givenAdminKeyMustBeValid(),
+        //                fridayThe13thSpec(),
+        //                updateDoesNotChangeBytecode(),
+        //                eip1014AddressAlwaysHasPriority(),
+        //                immutableContractKeyFormIsStandard(),
+        //                updateAutoRenewAccountWorks(),
+        //                updateStakingFieldsWorks());
     }
```

Comment:
need to be uncommented

Reply:
Yep, this was just a test/local change I pushed without my intention 😄 Will fix

Reply:
I have other nodes, just for test purposes, but I will fix them when I work on the suites(in this PR).